extensions/amp-list/0.1/amp-list.js

```diff
@@ -813,8 +815,12 @@ export class AmpList extends AMP.BaseElement {
       this.unlistenLoadMore_ = listen(
           loadMoreButtonClickable,
           'click', () => this.loadMoreCallback_());
+    }).then(() => {
       // Guarantees that the height accounts for the newly visible button
-      this.attemptToFit_(dev().assertElement(this.container_));
+      if (!this.loadMoreShown_) {
```

Comment:
We should only ever have this extra `attemptToFit` call once.

Reply:
if `attemptToFit` fails its attempt, would you still want to call it again later?

Reply:
If it fails the attempt, we will end up with the overflow situation, in which case we won't need to call it again.